wacom-gui/wacom_data.py

```diff
@@ -36,7 +37,7 @@ def __init__(self):
     def get_connected_tablets(self):
         # check if tablet is actually detected
         p = subprocess.Popen("xsetwacom --list devices", shell=True, stdout=subprocess.PIPE)
```

Comment:
@epifanio - phew, I am a little out of my depth with the whole wacom stuff - the oldest device I can test is Indous4. 
That said, please check your python version. If it is py3.7+, you might want to add `text=True` as a named param here. Also see my fist comment in the PR.
EL8 ships with 3.6; here Popen() does not need this param. Please report back if this works for you or not. If so, we'll need to refactor the *all* Popen() calls and check for python version.
Side note, you are really my favourite, python!